index.html

```diff
@@ -6,44 +6,75 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <script src="https://use.fontawesome.com/1fb0eb3889.js"></script>
+  <link rel="stylesheet" href="./stylesheets/style.css">
   <title>Where work happens | Slack</title>
 </head>
 
 <body>
   <nav>
-    <img src="./images/slack-logo.png" alt="slack_logo">
-    <ul>
-      <li>Why Slack?</li>
-      <li>Pricing</li>
-      <li>About Us</li>
-      <li>Your Workspaces</li>
-    </ul>
+    <div class="nav-div-img">
+      <img src="./images/slack-logo.png" alt="slack_logo" class="slack-logo">
+    </div>
+
+    <div class="nav-div-right">
+      <img src="./images/menu-alt-512.png" alt="" class="menu-img">
+      <!--
+      <ul>
+        <li>Why Slack?</li>
+        <li>Pricing</li>
+        <li>About Us</li>
+        <li>Your Workspaces</li>
+      </ul>
+      -->
+    </div>
   </nav>
+
   <header>
-    <img src="./images/home_talking.png" alt="home_work_happening">
     <h1>Where Work Happens</h1>
     <p>When your team needs to kick off a project, hire a new employee, deploy some code, review a sales contract, finalize next year's budget, measure an A/B test, plan your next office opening, and more, Slack has you covered.</p>
-    <button type="button" name="button">GET STARTED</button>
-    <p>Already using Slack? <a href="#">Sign in</a></p>
+      <form action="">
+        <input class="email" type="email" name="email" value="Email Adress">
+        <button type="button" name="button">GET STARTED</button>
+      </form>
+    <p class="p-signin">Already using Slack? <a href="#">Sign in</a></p>
+    <div class="header-div-img">
+      <img src="./images/home_talking.png" alt="home_work_happening">
+    </div>
   </header>
+  
   <main>
-    <div>
+    <div class="div-main">
```

Comment:
try to be more specific in the naming of your classes so that you can tell what each part does just by looking at the css.